src/components/PostDetails.tsx

```diff
@@ -98,21 +67,21 @@
       </div>
 
       <div className="block">
-        {!loaded && <Loader />}
+        {loaded && <Loader />}
```

Comment:
The condition here should be checking if the comments are still loading, so it should be `!loaded` instead of `loaded`. This is an error related to the task requirements.